src/test/java/com/meilisearch/integration/SearchTest.java

```diff
@@ -72,14 +71,14 @@ public void testBasicSearch() throws Exception {
     public void testSearchOffset() throws Exception {
         String indexUid = "SearchOffset";
         Index index = client.index(indexUid);
-        GsonJsonHandler jsonGson = new GsonJsonHandler();
 
         TestData<Movie> testData = this.getTestData(MOVIES_INDEX, Movie.class);
         Task task = index.addDocuments(testData.getRaw());
 
         index.waitForTask(task.getUid());
 
-        SearchRequest searchRequest = new SearchRequest("a").setOffset(20);
```

Comment:
This way do not work anymore? because of the `@setter`?

Reply:
If not you can use this option:
<img width="783" alt="image" src="https://user-images.githubusercontent.com/4116980/185424078-2b2c2b72-fff4-4eef-9706-6e2f027614da.png">
https://projectlombok.org/features/GetterSetter and make them a builder again! 🎉